modules/logging/structured/src/main/scala-3/tofu/logging/SingletonEnumLoggable.scala

```diff
@@ -0,0 +1,8 @@
+package tofu.logging
+
+import scala.deriving.Mirror
+
+object SingletonEnumLoggable:
```

Comment:
I missed one thing.

Could you also add `trait SignletonEnumLoggable[A] extends Loggable[A]`?

This would allow to use derives syntax

```
enum Foo derives SingletonEnumLoggable:
  case A
```

Reply:
hah, yep. current impl with `toString` doen't look solid

The proper way to get names is to use Mirror. Here is an example:
```scala
inline def nameOf[A](using m: scala.deriving.Mirror.Of[A]): String = scala.compiletime.constValue[m.MirroredLabel]

enum Sample:
   case X
   
nameOf[Sample] // "Sample"
nameOf[Sample.X] // "Sample.X"  
```